guides/common/modules/proc_creating-a-host.adoc

```diff
@@ -30,7 +30,12 @@ Alternatively, you can enter an address.
 * *Primary* -- Select this checkbox to use the DNS name from this interface as the host portion of the FQDN.
 * *Provision* -- Select this checkbox to use this interface for provisioning.
 This means TFTP boot will take place using this interface, or in case of image based provisioning, the script to complete the provisioning will be executed through this interface.
-Note that many provisioning tasks, such as downloading RPMs by *anaconda*, Puppet setup in a `%post` script, will use the primary interface.
+ifdef::orcharhino[]
+Note that many provisioning tasks, such as downloading packages by the operating system installer or Puppet setup in a `%post` script, will use the primary interface.
+endif::[]
+ifndef::orcharhino[]
+Note that many provisioning tasks, such as downloading RPMs by *anaconda* or Puppet setup in a `%post` script, will use the primary interface.
```

Comment:
```suggestion
Note that many provisioning tasks, such as downloading packages by *anaconda* or Puppet setup in a `%post` script, will use the primary interface.
```

Reply:
Would you go as far as to drop anaconda, i.e. drop the whole "if def orcharhino"? Else I'll just apply this.

Reply:
I'd prefer to keep *anaconda*. I am fine with keeping the ifdef here.

Reply:
Applied.